<launch>

  <include file="$(find speech_processing)/launch/followme.launch"/>
  <node name="NIHandtracker" pkg="gesture_detection" type="NIHandtracker" output="screen" >
    <remap from="points" to="/gesture/points"/>
  </node>

  <node name="main_state" pkg="main_state" type="main.py" output="screen" args="FollowGuiding" />
  <node pkg="rosbag" type="record" name="logging_follow_task" args="-o ../follow_guiding_task_bag/logging_follow_guide_bag nav_msgs/path /odom sensor_msgs/laserScan"  />

</launch>
